src/components/widgets/LineChart.js

```diff
@@ -24,8 +25,14 @@ const CustomLineChart = (props) => {
       />
       <XAxis
         dataKey={otherProps.xAxisKey}
+        domain={['dataMin', 'dataMax']}
+        tickFormatter={(unixTime) => format(unixTime, 'DD MMM YYYY')}
+        type='number'
```

Comment:
This now *assumes* that the x-axis is always a date, but as I think the Lab Trends is the only place we are using this widget, and react components will likely become deprecated, I think it's fine to just hard-code this change.